lib/persisters/local.js

```diff
@@ -66,6 +66,10 @@ class File {
 }
```

Comment:
^ you should use the async versions of read/write since this is all promise based